build/bin/set-kylin-home.sh

```diff
@@ -0,0 +1 @@
+export KYLIN_HOME=$KYLIN_HOME
```

Comment:
hi xiao, what's the behavior of this line? If KYLIN_HOME already been set, will export it again bring any change?